README.md

```diff
@@ -26,13 +26,17 @@ To run that demo on your own computer:
 
 ## How to Use
 
-Ensure packages are installed with correct version numbers by running:
+Ensure packages are installed with correct version numbers by running the following shell script: 
   ```sh
   (
     export PKG=react-dates;
     npm info "$PKG" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g; s/ *//g' | xargs npm install --save "$PKG"
   )
   ```
+If you're new to shell scripts, create an install.sh file, copy the command above to it and run by typing this in your terminal: 
```

Comment:
@savovs it looks like you're using cmder - this won't work with cmder (I tried that too - it also won't work in Cygwin)
I would suggest adding a note that you should use Git Bash for this if you're on Windows 
I'm not sure how close that is to BashOnWindows, but I have only verified with Git Bash